src/psyclone/tests/gocean1p0_opencl_test.py

```diff
@@ -272,17 +265,19 @@ def test_opencl_kernel_gen_wrong_kernel():
     # Test gen_ocl without any kernel argument
     with pytest.raises(GenerationError) as err:
         kschedule.gen_ocl()
-    assert ("GOcean Kernel should always have at least two arguments "
-            "representing the iteration indices.") in str(err)
+    assert ("GOcean 1.0 API kernels should always have at least two "
+            "arguments representing the iteration indices but the Symbol "
+            "Table for kernel 'test' has only '0' argument(s).") in str(err)
 
     # Test gen_ocl with 1 kernel argument
     kschedule.symbol_table.declare("arg1", "integer", [], "global_argument",
                                    True, False)
     kschedule.symbol_table.specify_argument_list(["arg1"])
     with pytest.raises(GenerationError) as err:
         kschedule.gen_ocl()
-    assert ("GOcean Kernel should always have at least two arguments "
-            "representing the iteration indices.") in str(err)
+    assert ("GOcean 1.0 API kernels should always have at least two "
+            "arguments representing the iteration indices but the Symbol "
+            "Table for kernel 'test' has only '1' argument(s).") in str(err)
```

Comment:
Please remove the quotes around the number of arguments.

Reply:
Done